behat.yml

```diff
@@ -18,5 +18,12 @@ default:
           base_url: "http://gigadb.dev/"
           goutte: ~
           selenium2:
-            wd_host: http://phantomjs:8910/wd/hub
+            wd_host: "http://chrome:4444/wd/hub"
+            browser: chrome
+            capabilities:
+              chrome:
+                switches:
+                  - "--headless"
+                  - "--disable-gpu"
+                  - "--window-size=1024,1024"
```

Comment:
This line conflicts with the `develop` branch which has window-size=1024,768. Your tests in `admin-file.feature` pass with this window size so I think we should stick to 1024,768 unless there is a reason to have it as 1024,1024?

Reply:
The screen size of `window-size=1024,768` is not enough to capture the `attribute` row which shows the `Delete` button.

Reply:
OK, having a common window size of 1920,1080 seems more appropriate then.